main.yml

```diff
@@ -4,6 +4,23 @@
 # Note, 'local' (the host name in the inventory)
 #   should be assigned ``ansible_connection=local``.
 
+# Ansible requires python2, while Ubuntu >=15 strictly uses Python3.
```

Comment:
I with draw my comment. ISTR I did in fact select something in the 'tasksel' setp of the install, I bet that dragged it in.